.github/workflows/docker.yml

```diff
@@ -39,11 +38,17 @@ jobs:
       - name: Set tag variables
         id: setTagVariables
         run: |
-          echo "version=$(git describe)" >> $GITHUB_OUTPUT
+          if [[ ${{ github.ref_type }} == tag ]]; then
+            echo "version=stable-${{ github.ref_name }}" >> $GITHUB_OUTPUT
+          else
+            SHORT_SHA=$(git rev-parse --short HEAD)
+            BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
will this work? if github is checking out a commit number on the agent, this will return HEAD, also why not to use wd_branch ?

Reply:
HEAD is fine